docs/docsite/rst/user_guide/playbooks_variables.rst

```diff
@@ -18,7 +18,9 @@ Once you understand the concepts and examples on this page, read about :ref:`Ans
 Creating valid variable names
 =============================
 
-Not all strings are valid Ansible variable names. A variable name must start with a letter, and can only include letters, numbers, and underscores. `Python keywords`_ and :ref:`playbook keywords<playbook_keywords>` are not valid variable names.
+Not all strings are valid Ansible variable names. A variable name can only include letters, numbers, and underscores. `Python keywords`_ and :ref:`playbook keywords<playbook_keywords>` and :ref:`playbook keywords<playbook_keywords>` are not valid variable names. A variable name cannot begin with a number.
+
+Variable names can begin with an underscore. In many programming languages, variables that begin with an underscore are private. This is not true in Ansible. Variables that begin with an underscore are treated exactly the same as any other variable. Do not rely on this convention for privacy or security.
```

Comment:
Is there some place in the documentation for people who are developing Collections or Roles?  I'd like to add a note there about the idiomatic use of Collection or Role variables beginning with an underscore to indicate that the variable is "protected" or "private" to the Collection or Role and should not be used outside of the Collection or Role.